assets/src/blocks/amp-story/constants.js

```diff
@@ -0,0 +1,5 @@
+
+export const IMAGE_BACKGROUND_TYPE = 'image';
+export const VIDEO_BACKGROUND_TYPE = 'video';
+export const ALLOWED_MEDIA_TYPES = [ 'image', 'video' ];
+export const POSTER_ALLOWED_MEDIA_TYPES = [ 'image' ];
```

Comment:
For some reason these don't seem to work for me, for example I'm still able to upload `.pdf` files, etc. Do they work as expected for you?

Reply:
Looking at the `MediaUpload` component's implementation, you are able to upload anything, but not select non-allowed media types. Also, the `allowedTypes` property filters existing items in the media modal, so that e.g. PDFs are never shown to you in the modal.

Not sure if we can easily prevent upload of non-allowed media types. Don't feel like it's really needed though.